site/content/events/2016-toronto/_cfp_date_announce.txt

```diff
@@ -1 +1 @@
-Announce date
+February 15, 2016
```

Comment:
Hey @WhatsChrisDoing - do you intend for selected proposals to be announced the day after submissions close?